components/org.wso2.carbon.identity.oauth.uma.resource.service/src/main/java/org/wso2/carbon/identity/oauth/uma/resource/service/ResourceConstants.java

```diff
@@ -21,14 +21,22 @@
  */
 public class ResourceConstants {
 
+    public static final String RESOURCE_PATH =
+            "https://localhost:9443/api/identity/oauth2/uma/resourceregistration/v1.0";
```

Comment:
localhost and port hard coded? Use the IdentityUtil.getServerURL() to get the server url.